video_xblock/exceptions.py

```diff
@@ -27,6 +27,13 @@ def __str__(self):
         """
         return self.detail
 
+    @property
+    def message(self):
```

Comment:
Do we really need it?

Reply:
in python3 built-in Exception doesn't have the attribute `message` like it was in python2, but in the codebase these are many places where that attribute is used, I decided to add one to avoid refactoring of many places of the codebase.

Reply:
You r lazy one, do u know that???

Reply:
Maybe, but so patient to wait for your review